spyder_kernels/comms/kernelcomm.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+In addition to the remote_call mechanism implemented in CommBase:
+ - Send a message to a debugging kernel
+"""
+import logging
+
+from qtpy.QtCore import QEventLoop, QObject, QTimer, Signal
```

Comment:
This is really problematic. This package should not depend on Qt because it would make it quite heavy and difficult to install.

Since this adds new functionality (i.e. sending a message to a debugging kernel), I'd say we should leave this for another PR for now, so we can discuss about it later.

Reply:
This is the frontend -> kernel communication file. I will move that to spyder. Where should I put it?